ast/misc.go

```diff
@@ -721,6 +721,40 @@ func (n *SetPwdStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+type ChangeStmt struct {
+	stmtNode
+
+	NodeType  string
+	State     string
+	IpAndPort string
+}
+
+// Restore implements Node interface.
+func (n *ChangeStmt) Restore(ctx *RestoreCtx) error {
+	ctx.WriteKeyWord(fmt.Sprintf("CHANGE %s TO", strings.ToUpper(n.NodeType)))
+	ctx.WriteKeyWord(fmt.Sprintf("%s_STATE", strings.ToUpper(n.NodeType)))
```

Comment:
```suggestion
	ctx.WriteKeyWord(n.NodeType + "_STATE")
```

Though this shows the syntax is pretty redundant, why not just use `CHANGE PUMP TO STATE = ` / `CHANGE DRAINER TO STATE = `?

Reply:
if i use 'state' as keyword ,it will report an error.
After discussions with your colleagues, i use 'node_state' install.